python/src/XMLStorageManager_doc.i.in

```diff
@@ -4,10 +4,14 @@
 Available constructors:
     XMLStorageManager(*filename*)
 
+    XMLStorageManager(*filename, compressionLevel*)
+
 Parameters
 ----------
 fileName : str
     Name of the file containing the study.
+compressionLevel : int
+    Compression level of the resulting XML file, in the range 0,...,9 (0=no compression, 9=best compression). The compression is done only if the XML lib has been compiled with zlib support. If not given, the compression level is equal to :class:`~openturns.ResourceMap.GetAsUnsignedInteger('XMLStorageManager-DefaultCompressionLevel')` with a default value of zero.
```

Comment:
line too long,
you can only mention the resourcemap key name, not :class:`~openturns.ResourceMap.GetAsUnsignedInteger('XMLStorageManager-DefaultCompressionLevel')`